registry_module_integration_test.go

```diff
@@ -446,6 +446,158 @@ func TestRegistryModulesCreateVersion(t *testing.T) {
 	})
 }
 
+func TestRegistryModulesShowVersion(t *testing.T) {
+	skipUnlessBeta(t)
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgTest, orgTestCleanup := createOrganization(t, client)
+	defer orgTestCleanup()
+
+	registryModuleTest, registryModuleTestCleanup := createRegistryModule(t, client, orgTest, PrivateRegistry)
+	defer registryModuleTestCleanup()
+
+	t.Run("when the version exists", func(t *testing.T) {
+		options := RegistryModuleCreateVersionOptions{
+			Version: String("1.2.7"),
+		}
+
+		registryModuleIDTest := RegistryModuleID{
+			Organization: orgTest.Name,
+			Name:         registryModuleTest.Name,
+			Provider:     registryModuleTest.Provider,
+		}
+
+		rmv, err := client.RegistryModules.CreateVersion(ctx, registryModuleIDTest, options)
+		require.NoError(t, err)
+		assert.NotEmpty(t, rmv.ID)
+		assert.Equal(t, *options.Version, rmv.Version)
+
+		rmvRead, errRead := client.RegistryModules.ReadVersion(ctx, registryModuleIDTest, *options.Version)
+
+		require.NoError(t, errRead)
+		assert.NotEmpty(t, rmvRead.ID)
+
+		t.Run("relationships are properly decoded", func(t *testing.T) {
+			assert.Equal(t, registryModuleTest.ID, rmvRead.RegistryModule.ID)
+		})
+
+		t.Run("timestamps are properly decoded", func(t *testing.T) {
+			assert.NotEmpty(t, rmvRead.CreatedAt)
+			assert.NotEmpty(t, rmvRead.UpdatedAt)
+		})
+
+		t.Run("links are properly decoded", func(t *testing.T) {
+			assert.NotEmpty(t, rmvRead.Links["upload"])
+			assert.Contains(t, rmvRead.Links["upload"], "/object/")
+		})
+	})
+
+	t.Run("when the version does not exist", func(t *testing.T) {
```

Comment:
I was initially confused by the heading, since the`RegistryModuleCreateVersionOptions` that follows this line does carry a version, but I soon realized that the summary referred to the `Read` action, not the `Create` action. I would suggest renaming test summary to something like:
```suggestion
	t.Run("when reading a version that does not exist", func(t *testing.T) {
```